docs/releases/1.10.txt

```diff
@@ -832,6 +832,28 @@ features, is deprecated. Replace it with a custom lookup::
     models.CharField.register_lookup(Search)
     models.TextField.register_lookup(Search)
 
+Using ``request.user.is_authenticated()`` and ``request.user.is_anonymous()`` as methods
+----------------------------------------------------------------------------------------
+
+The following members of
+:class:`~django.contrib.auth.models.AbstractBaseUser` and
+:class:`~django.contrib.auth.models.AnonymousUser` classes are now
+properties, and should no longer be invoked as methods:
```

Comment:
We should explain the rationale for the change to avoid criticism for useless deprecations.